manager/controllers/packages.go

```diff
@@ -15,6 +15,7 @@ var PackagesOpts = ListOpts{
 	// By default, we show only fresh systems. If all systems are required, you must pass in:true,false filter into the api
 	DefaultFilters: map[string]FilterData{},
 	DefaultSort:    "name",
+	SecondarySort:  "name",
```

Comment:
`name` is also no unique. And this also joins `package` table so `package.id` could be good one.